piet-cairo/src/lib.rs

```diff
@@ -309,7 +298,13 @@ impl<'a> RenderContext for CairoRenderContext<'a> {
         dst_rect: impl Into<Rect>,
         interp: InterpolationMode,
     ) {
-        draw_image(self, image, Some(src_rect.into()), dst_rect.into(), interp);
+        draw_image(
```

Comment:
it's strange to me that this isn't just a method on `CairoImageContext` itself, but that can be out of scope for this PR.

Reply:
I've moved it to be a method on `CairoRenderContext`.